templates/python3/action-{{action_name}}.py.tpl

```diff
@@ -1,29 +1,39 @@
 #!/usr/bin/env python3
-# -*- coding: utf-8 -*-
 
-import configparser
+import toml
 from hermes_python.hermes import Hermes
 from hermes_python.ffi.utils import MqttOptions
 from hermes_python.ontology import *
-import io
 
 CONFIGURATION_ENCODING_FORMAT = "utf-8"
 CONFIG_INI = "config.ini"
-
-class SnipsConfigParser(configparser.SafeConfigParser):
-    def to_dict(self):
-        return {section : {option_name : option for option_name, option in self.items(section)} for section in self.sections()}
-
+PLATFORM_CONFIG_TOML = '/etc/snips.toml'
 
 def read_configuration_file(configuration_file):
     try:
-        with io.open(configuration_file, encoding=CONFIGURATION_ENCODING_FORMAT) as f:
-            conf_parser = SnipsConfigParser()
-            conf_parser.readfp(f)
-            return conf_parser.to_dict()
-    except (IOError, configparser.Error) as e:
+        with open(configuration_file, encoding=CONFIGURATION_ENCODING_FORMAT) as f:
+            return toml.loads(f.read())
+    except (OSError, ValueError) as e:
```

Comment:
in python3 IOError is part of OSError and when the file does not exist it will throw a FileNotFoundError, which is no IOError, but a OSError, so it makes more sence to catch OSError.
In python3 io.open is the same as open (not in python2)

Reply:
partially fixed the issue #10 since it just assumes it is a toml file, which will work with ini aswell

Reply:
using toml.load now so it is not required anymore